package.json

```diff
@@ -34,7 +34,7 @@
     "terser:bench": "terser -c passes=2 -m --sequences 40 --module build/rollup/benchmark.umd.js -o build/benchmark-bundle.min.js",
     "headers": "node ./tools/header.js build/peggy.min.js build/benchmark-bundle.min.js build/test-bundle.min.js",
     "deploy": "npm run deploy:peggy && npm run deploy:tests && npm run deploy:bench",
-    "coverage:bin": "PEGGY_CLI_DEBUG=1 npm run rollup && nyc --reporter lcov jest run.spec.ts",
+    "coverage:bin": "PEGGY_CLI_DEBUG=1 npm run rollup && jest run.spec.ts",
```

Comment:
Make sure not to check in the sourceMap'd version, please.

Reply:
jest doesn't follow fork/exec's when recording coverage.  the `nyc` wrapper does.  With this refactor, most of the tests don't need to fork/exec, so `nyc` isn't needed anymore.